### PR TITLE
PHP 8.0

### PR DESCRIPTION
**Pull request by sandbergja**
related to https://github.com/pulibrary/princeton_ansible/issues/3350

### CODE REVIEW

File: sites/all/libraries/CAS/CAS.php [hunk 1]
```diff
@@ -20,7 +20,7 @@
  *
  *
  * Interface class of the phpCAS library
- * PHP Version 5
+ * PHP Version 7
```
Comment (christinach) [inline sites/all/libraries/CAS/CAS.php:23]: https://github.com/apereo/phpCAS download the most recent release and copy it into place.

File: .lando.yml [hunk 2]
```diff
@@ -1,9 +1,10 @@
 name: library-main
 recipe: drupal7
 config:
-  php: '7.4'
+  php: '8.0'
   webroot: .
   database: mariadb
+  drush: '8.4.11'
```
Comment (christinach) [inline .lando.yml:7]: the default version of drush doesnt work with lando. had to use specific version of drush.

File: sites/all/libraries/package.xml [hunk 3]
```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (christinach) [inline sites/all/libraries/package.xml:1]: This was an automated file when upgrading the php CAS library.

**Review by christinach — APPROVED** [review]

File: sites/all/libraries/CAS/CAS.php (hunk 1 above)
Comment (sandbergja) [reply sites/all/libraries/CAS/CAS.php:23]: Thanks!  I added these procedures to doc/phpCAS.md in this PR.

**Pull request merged**